webpack.config.js

```diff
@@ -2,11 +2,9 @@
 // See LICENSE.txt for license information.
 
 const childProcess = require('child_process');
-
 const path = require('path');
 
```

Comment:
nit : since we are doing those in L9 ,L5

Reply:
I think I messed up what I was trying to do here. I wanted to group the Node libraries, so I do want a blank line, but it should be after `url` instead of here. I'll change that